web/src/app/modules/shared/components/form-view-container/form-view-container.component.spec.ts

```diff
@@ -82,5 +84,38 @@ describe('FormViewContainerComponent', () => {
       expect(element.querySelector('cds-radio-group')).not.toBeNull();
       expect(element.querySelector('cds-radio')).not.toBeNull();
     });
+
+    it('should create a select', () => {
+      component.form = {
+        fields: [
+          {
+            config: {
+              configuration: {
+                choices: [
+                  { label: 'a', value: 'a', checked: true },
+                  { label: 'b', value: 'b', checked: false },
+                  { label: 'c', value: 'c', checked: false },
+                ],
+              },
+              label: 'label',
+              name: 'name',
+              type: 'select',
+              value: null,
+              placeholder: '',
+              error: null,
+              validators: null,
+            },
+            metadata: { type: 'formField' },
+          },
+        ],
+      };
+      component.formGroup = formHelper.createFromGroup(
+        component.form,
+        formBuilder
+      );
+      fixture.detectChanges();
+
+      expect(element.querySelector('cds-select')).not.toBeNull();
```

Comment:
Since choices are added within the test, it would be reasonable to check if the input choices are generated as expected.